osi_lane.proto

```diff
@@ -1034,6 +1034,18 @@ message LaneBoundary
             // A structure (e.g. building or tunnel wall).
             //
             TYPE_STRUCTURE = 13;
+
+            // A barrier (e.g. a concrete barrier on a highway).
+            //
+            TYPE_BARRIER = 14;
```

Comment:
Should probably also be described in more detail to distinguish from guard_rail, structure and sound_barrier?